concepts/05 Widgets/Form/01 Getting Started with Form/70 Validate the Form Data.md

```diff
@@ -0,0 +1,224 @@
+DevExtreme includes a validation engine that checks edited values before they are saved. This engine supports different validation rule types, such as [Email](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/EmailRule/), [Numeric](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/NumericRule/), [Pattern](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/PatternRule/), and more. 
+
+To apply validation rules to a simple item, specify them in the [validationRules[]](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#validationRules) array. The Form validates an editor when this editor's value changes. To validate all editors simultaneously, call the [validate()](/Documentation/ApiReference/UI_Widgets/dxForm/Methods/#validate) method. 
+
+You can specify an item's [isRequired](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#isRequired) property to implicitly apply the [Required Rule](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/RequiredRule/) to it. The required items have red asterisk marks.
+
+In the following example, we specify the [isRequired](Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#isRequired) property for the `Name` item. We also specify the **NumericRule** and **EmailRules** for two other items. The [showValidationSummary](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#showValidationSummary) property is set to **true**.
```

Comment:
> two other items

may be "NumericRule for officeNumber and EmailRules for 'name???'"

email rule:
```
                dataField: "name",
                validationRules: [{
                    type: "email",
                    message: "This is not a valid Email"
                }] 
```

Reply:
Thanks, good point.